File: Back_end_barbearia_app/Models/Agendamento.cs
using System;

namespace Back_end_barbearia_app.Models
{
    public class Agendamento
{
    public int Id { get; set; }
    public int ClienteId { get; set; }
    public int BarbeiroId { get; set; }
    public int ServicoId { get; set; }
    public DateTime DataHora { get; set; }
    public string Status { get; set; } = "pendente"; // "pendente", "concluido", "cancelado"
}
}

File: Barbearia-App/app/src/main/java/com/example/appbarbearia/models/Agendamento.kt
package com.example.appbarbearia.models

import kotlinx.serialization.Serializable

@Serializable
data class Agendamento(
    val id: Int? = null,
    val clienteId: Int,
    val barbeiroId: Int,
    val servicoId: Int,
    val dataHora: String,
    val clienteNome: String? = null,
    val barbeiroNome: String? = null,
    val servicoNome: String? = null
)
